public/index.html

```diff
@@ -0,0 +1,75 @@
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css" integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ==" crossorigin=""/>
+  <!-- Make sure you put this AFTER Leaflet's CSS -->
+<script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+crossorigin=""></script>
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+<!-------------------------------Header  ------------------------------------>
+<div id='header'>
+  <h1>Yelpington!</h1>
+</div>
+<div id="mainBox">
+<!--------------------------------Side Navigation Bar  ----------------------->
+<div id='sideNav'>
+
+</div>
+<!------------------------------- Map ---------------------------------------->
+<div id="myMap">
+
+</div>
+<!--------------------------------Footer   --------------------------------->
+</div>
+<div id='footer'>
+    <section id="footer">
+        <div class="headerQC">
+          <h3 class="QC">QUESTIONS AND COMMENTS? <b>GET IN TOUCH</b></h3>
+        </div>
+        <div class="containerQC">
+          <div class="colunm1">
+            <div class="nameQC emailQC">
+              <input id="input-name" type="text" placeholder="Name">
+              <input id="input-email" type="text" placeholder="Email">
+            </div>
+            <div class="commentQC">
+              <textarea name="comment section" id="comment-area" placeholder="    Comment"></textarea>
+            </div>
+            <div class="buttonQC">
+              <a class="learn-more-button"><i class="fas fa-envelope"></i> Send Message</a>
+            </div>
+          </div>
+          <div class="column2">
+            <div class="contentQC">
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Consectetur autem placeat libero enim quibusdam
+                deleniti.</p>
+            </div>
+            <div class="addresses">
+              <div class="addressStack1">
+                <div class=stackItem><i class="fab fa-twitter"></i> Twitter</div>
+                <div class=stackItem><i class="fab fa-facebook"></i> Facebook</div>
+                <div class=stackItem><i class="fas fa-envelope-square"></i> Email</div>
+                <div class="stackItem"><i class="fas fa-tree"></i>webaddress.com</div>
+  
+              </div>
+              <div class="addressStack2">
+                <div class=stackItem><i class="fas fa-home"></i> 182 Burlingotn Vermont 05401</div>
+                <div class=stackItem><i class="fas fa-phone"></i> 802-888-2222</div>
+                <div class=stackItem><i class="fab fa-instagram"></i> Instragram</div>
+              </div>
+            </div>
+          </div>
+      </section>
+</div>
+  <script src="main.js"></script>
+  <script src="https://kit.fontawesome.com/66951b5047.js" crossorigin="anonymous"></script>
```

Comment:
Do this before you link to your stylesheet (line 13) to make sure you have access to these fonts there.